Documentation~/Blender.md

```diff
@@ -69,3 +70,27 @@ This option will only appear if we install [ProBuilder](https://docs.unity3d.com
 > If **Bake Modifiers** option is enabled in the plugin, 
 > a mesh that is sent from Unity back into Blender will have modifiers applied beforehand and have them removed after.
 
+## Material sync mode
+
+The MeshSyncServer in Unity always creates a material with a default shader for the active render pipeline.
+There are multiple options to export materials from Blender:
+
+|**Mode** |**Description** |
+|:---       |:---|
+| None | Only the material names are exported to Unity and default materials created. Users can override the materials in the mapping on the MeshSyncServer to use their own materials. |
+| Basic | The materials are synced to Unity in a limited way. Refer to [Basic Material sync mode](#Basic-Material-sync-mode) for more details.||
+
+## Basic Material sync mode
+MeshSync looks for an active material output node and exports the BSDF connected to that. 
+Colors and textures assigned to the active BSDF are exported.
+Smoothness and metallic are baked into maps required by Unity depending on the active render pipeline.
```

Comment:
Nice rewrite. 
I think it may be better to explicitly mention that we support built-in / URP / and HDRP materials.

Reply:
Done